PrototopeJSBridge/GeometryBridge.swift

```diff
@@ -173,6 +174,31 @@ import Prototope
 		self.rect = rect
 		super.init()
 	}
+    
+    public func inset(args: NSDictionary) -> RectJSExport {
+        let hasKey = { (key: String) -> Bool in
+            return (args.objectForKey(key) as? Double) != nil
+        }
+        let getValue = { (key: String) -> Double in
+            return (args.objectForKey(key) as! Double?) ?? 0
+        }
+        
+        let justValue = hasKey("value")
+        let simplified = hasKey("vertical") || hasKey("horizontal")
+        let core = hasKey("top") || hasKey("right") || hasKey("bottom") || hasKey("left")
+        
+        switch (justValue, simplified, core) {
```

Comment:
What do you think about using ifs here instead? This way, you can avoid the truth-table switch statement, which might make the intention a little clearer:

```swift
if let justValue = hasKey("value") {
    return RectBridge(rect.inset(value: getValue("value")))
}

if let simplified = hasKey("vertical") || hasKey("horizontal") {
    return ...
}
// ...

Environment.currentEnvironment?.exceptionHandler("Trying to call inset on with invalid parameters: \(args)")
return self
```